<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PravekshaUIClass</class>
 <widget class="QMainWindow" name="PravekshaUIClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>836</width>
    <height>540</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Praveksha</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>821</width>
      <height>461</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tbGeneral">
     <attribute name="title">
      <string>General</string>
     </attribute>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>801</width>
        <height>25</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="btnStartCallibration">
         <property name="text">
          <string>Start Callibration</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnStopCallibration">
         <property name="text">
          <string>Stop Callibration</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="btnStat">
         <property name="text">
          <string>Refresh Stat</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QListView" name="lstLog">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>341</width>
        <height>361</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Violations :</string>
      </property>
     </widget>
     <widget class="QLabel" name="lblImage">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>60</y>
        <width>441</width>
        <height>251</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tbStat">
     <attribute name="title">
      <string>Statistics</string>
     </attribute>
     <widget class="QWidget" name="widgetGraph" native="true">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>30</y>
        <width>451</width>
        <height>361</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>20</y>
        <width>261</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Statistics for the past 24 hours</string>
      </property>
      <property name="textFormat">
       <enum>Qt::RichText</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>90</y>
        <width>171</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Total Number of Vehicles:</string>
      </property>
     </widget>
     <widget class="QLCDNumber" name="txtTotVehicles">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>90</y>
        <width>64</width>
        <height>23</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="numDigits">
       <number>4</number>
      </property>
      <property name="value" stdset="0">
       <double>0.000000000000000</double>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>210</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
        <weight>75</weight>
        <italic>false</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>1 - Wrong Lane</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>250</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>2 - High Speed</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>290</y>
        <width>201</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>3 - High Speed and Wrong Lane</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>180</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Legend</string>
      </property>
     </widget>
     <widget class="QLCDNumber" name="txtWrongLane">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>210</y>
        <width>64</width>
        <height>23</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="numDigits">
       <number>4</number>
      </property>
     </widget>
     <widget class="QLCDNumber" name="txtHighSpeed">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>250</y>
        <width>64</width>
        <height>23</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="numDigits">
       <number>4</number>
      </property>
     </widget>
     <widget class="QLCDNumber" name="txtBoth">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>290</y>
        <width>64</width>
        <height>23</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="numDigits">
       <number>4</number>
      </property>
      <property name="segmentStyle">
       <enum>QLCDNumber::Filled</enum>
      </property>
     </widget>
     <widget class="Line" name="line">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>130</y>
        <width>321</width>
        <height>16</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="Line" name="line_2">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>20</y>
        <width>20</width>
        <height>381</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tbSettings">
     <attribute name="title">
      <string>Settings</string>
     </attribute>
     <widget class="QPushButton" name="btnSetLoggerPath">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>71</y>
        <width>41</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Set</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnSetDistance">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>41</y>
        <width>41</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Set</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>41</y>
        <width>46</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Distance</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>71</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Logger Path</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtLoggerPath">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>71</y>
        <width>321</width>
        <height>20</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Logger Path</string>
      </property>
      <property name="text">
       <string>F:/20july2013_time_7am_9am/Logs/</string>
      </property>
      <property name="cursorPosition">
       <number>32</number>
      </property>
     </widget>
     <widget class="QPushButton" name="btnBrowse">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>11</y>
        <width>21</width>
        <height>23</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>8</pointsize>
        <stylestrategy>PreferDefault</stylestrategy>
        <kerning>true</kerning>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string>...</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtDistance">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>41</y>
        <width>111</width>
        <height>20</height>
       </rect>
      </property>
      <property name="whatsThis">
       <string>Length of the mapping region</string>
      </property>
      <property name="text">
       <string>50</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtPath">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>11</y>
        <width>491</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>F:/20july2013_time_7am_9am/Export_2013-07-24_114637/0 - 2013-07-20 12-00-00-033.asf</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label1_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>51</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Video Path</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>836</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="pravekshaui.qrc"/>
 </resources>
 <connections/>
</ui>
